src/main/java/gov/cancer/pageobject/section/FeatureCard.java

```diff
@@ -0,0 +1,179 @@
+    this.card = element;
+
+    title = ElementHelper.findElement(card, TITLE_SELECTOR);
+    description = ElementHelper.findElement(card, DESCR_SELECTOR);
+
+    WebElement im = ElementHelper.findElement(card, IMAGE_SELECTOR);
+    image = new Image(im);
+
+    List<WebElement> links = ElementHelper.findElements(card, LINK_SELECTOR);
+    cardLink = links.get(0);
+
+    if (links.size() > 1) {
+      exitNotificationLink = links.get(1);
+    }
+
+  }
+
+  /**
+   * Retrieve the card's promotional image or override.
+   * 
+   * @return An Image object containing the card's promotional image.
+   */
+  public Image getImage() {
+    return this.image;
+  }
+
+  /**
+   * Retrieve the card's title.
+   * 
+   * @return String containing the title text.
+   */
+  public String getTitle() {
+    return this.title.getText();
+  }
+
+  /**
+   * Retrieve the card's description.
+   *
+   * @return String containing the description if present, null otherwise.
+   */
+  public String getDescription() {
+    if (description != null)
+      return description.getText();
+    else
+      return null;
+  }
+
+  /**
+   * Retrieve the link the card goes to.
+   *
+   * @return A Link instance representing the link.
+   */
+  public String getLink() {
+    return cardLink.getAttribute("href");
+  }
+
+  /**
+   * Check the card title is correct.
+   *
+   * @return true if card title is correct.
+   */
+  public boolean isTitleCorrect(String cardtitle) {
+    if (getTitle().matches(cardtitle)) {
+      return true;
+    } else
+      return false;
+  }
+
+  /**
+   * Check the card description is correct.
+   *
+   * @return true if card description is correct.
+   */
+  public boolean isDescCorrect(String carddesc) {
+    if (getDescription() == null && carddesc == "") {
+      return true;
+    } else if (getDescription().startsWith(carddesc)) {
+      return true;
+    } else
+      return false;
+  }
+
+  /**
+   * Check the image is correct.
+   *
+   * @return true if image is correct.
+   */
+  public boolean isImageCorrect(String source) {
+    if (getImage().getUrl().getPath().matches(source)) {
+      return true;
+    } else
+      return false;
+  }
+
+  /**
+   * Check the image alt text is correct.
+   *
+   * @return true if image alt text is correct.
+   */
+  public boolean isAltTextCorrect(String alt) {
+    if (getImage().getAltText().matches(alt)) {
+      return true;
+    } else
+      return false;
+  }
+
+  /**
+   * Check the card link is correct.
+   *
+   * @return true if card link is correct.
+   */
+
+  public boolean isCardLinkCorrect(String link) {
+    if (getLink().contains(link)) {
+      return true;
+    } else
+      return false;
+  }
+
+  /**
+   * Check the exit notification is present for External feature cards.
+   *
+   * @return true if exit notification is present.
+   */
+  public boolean isExitNotificationPresent() {
+    return exitNotificationLink.getAttribute("title").contains("Exit Disclaimer");
+  }
```

Comment:
This has multiple problems.

* It will fail on a Spanish page.
* The page class should only return values, it should not attempt to evaluate them.
* Instead of using the text to determine whether the exit notification is present, it should look for the `icon-exit-notification` class on the `<a>` tag.
* This should be replaced by a method which returns a helper object representing a tag. The tag object would in turn expose methods for retrieving the value of various attributes which the **test** class would then evaluate for correctness.